packages/testcontainers/src/chains/reg_test_container/spv.ts

```diff
@@ -0,0 +1,31 @@
+import { RegTestContainer } from './'
+
+/**
+ * SPV testing functionnalities
+ */
+export class Spv {
+  private readonly client: RegTestContainer
+
+  constructor (client: RegTestContainer) {
+    this.client = client
+  }
+
+  /**
+   * Funds a Bitcoin address with 1 BTC.
```

Comment:
maybe mentioning this as "testing purpose only" is a good idea?
```suggestion
   * Funds a Bitcoin address with 1 BTC(for test purposes only)
```